openlabcmd/openlabcmd/cli.py

```diff
@@ -11,6 +12,9 @@
 from openlabcmd.utils import _color
 from openlabcmd import zk
 
+from zuul.driver.github.githubconnection import GithubConnection
+from zuul.driver.github import GithubDriver
```

Comment:
Don't put much work in `cli.py`. IMO, cli.py is just used for cmd interface. I suggest to create a new file for github task.  so that other tools can reuse it as a library if needed.

Reply:
> Don't put much work in cli.py. IMO, cli.py is just used for cmd interface. I suggest to create a new file for github task. so that other tools can reuse it as a library if needed.

Sure, will do.

> And it's strange that openlabcmd rely on zuul. How about using PyGithub lib directly?
The /app/installations is not supported by PyGithub, and as we F2F discussion, I will keep this import, but need to move it into method with import error exception process.